docs/releasing/publishing.md

```diff
@@ -75,21 +75,25 @@ Developers should pair on releases. When remote working, it can be useful to be
 3. Run `npm run publish-release`, which will prompt you to check whether the npm tag looks as expected.
 
   If you're following these instructions, you're probably doing a sequential release, meaning
-  the tag should be 'latest'.
+  the tag should be 'latest'. This step will also create a ZIP file containing the entirety of the release in the root
+  of your local instance of govuk-frontend and will be referenced by the command line in its prompt ("...and attach this").
```

Comment:
It feels to me like this should be two paragraphs, though I realise having a paragraph about the ZIP generation might be a bit awkward when both the previous and next paragraphs are about npm tags.

Reply:
I think this should probably go with/after step 4. At this point, pressing `y` simply means accepting that the tag looks correct - it won't publish the release until you press `y` again as prompted to in step 4.

Reply:
I've moved this into its own paragraph and moved it under what is now step 5.